src/corvus_python/pyspark/utilities/session_utils/get_or_create_spark_session.py

```diff
@@ -1,20 +1,27 @@
 """Copyright (c) Endjin Limited. All rights reserved."""
 
-from pyspark.sql import SparkSession
 import os
+
+try:
+    from pyspark.sql import SparkSession
+except ImportError as exc:
+    raise ImportError("PySpark is required for get_or_create_spark_session functionality. "
+                      "Install using corvus-python[pyspark]") from exc
+
 from .local_spark_session import LocalSparkSessionConfig, LocalSparkSession
 
 
 def get_or_create_spark_session(
         local_spark_session_config: LocalSparkSessionConfig = None
-        ) -> SparkSession:
+        ):
```

Comment:
Now that's cheating :)

Reply:
Ah I thought I caught all those. That can get added back in.